packages/buttons/src/_group.css

```diff
@@ -1,137 +0,0 @@
-/* stylelint-disable no-descending-specificity, selector-max-specificity */
-@import '@zendeskgarden/css-variables';
-
-:root {
-  --zd-btn-group--light-border-color: var(--zd-color-grey-100);
-  --zd-btn-group--primary-focused-box-shadow:
-    inset 0 0 0 3px
-    color-mod(var(--zd-color-white) alpha(35%));
-  --zd-btn-group--primary-line-height: var(--zd-btn--icon-size);
-  --zd-btn-group--primary--sm-line-height: var(--zd-btn--icon--sm-size);
-  --zd-btn-group--primary--lg-line-height: var(--zd-btn--icon--lg-size);
-}
```

Comment:
Could you go into detail the `group` styling removal? You mentioned

> for required accessibility logic

but, I think the only required logic is `role="group"` on the containing element.

Reply:
Clearly there is more to the React component than `role`. See https://github.com/zendeskgarden/react-components/blob/master/packages/buttons/src/elements/ButtonGroup.tsx for controlling selection and default selection. We don't want to carry that weight (JS required) over here.

Reply:
> Clearly there is more to the React component than role

Agreed. But, those additions, including roving-tab-index, aren't required a11y. Only `[role="group", aria-label="some label"]` are required from my understanding.

> We don't want to carry that weight over here

Agreed. We historically haven't supported those interactions in this package, I don't think now is the time to make that change.